restic/internal/key_test.py

```diff
@@ -0,0 +1,56 @@
+import unittest
```

Comment:
What do you mean by thing? 🤔 The import statemet is the same as for any other test.

Reply:
Oh, sorry. It's not about this particular line. I just meant that we should exercise `restic.key` in `e2e/test.py`. There's not a good place for me to place that note since the PR doesn't touch `e2e/test.py`, so I made it on the top of `key_test.py`, but the comment is unrelated to this particular import line.

Can we add an end-to-end test in `e2e/test.py`?

Reply:
Ah, gotcha. I've added `restic.key.list()`.